src/main/resources/META-INF/plugin.xml

```diff
@@ -279,7 +279,8 @@
         <!-- LSP textDocument/definition request support -->
         <gotoDeclarationHandler
                 id="LSPGotoDeclarationHandler"
-                implementation="com.redhat.devtools.lsp4ij.features.navigation.LSPGotoDeclarationHandler"/>
+                implementation="com.redhat.devtools.lsp4ij.features.navigation.LSPGotoDeclarationHandler"
+                order="first"/>
```

Comment:
Made this first to ensure that its logic is processed before other `gotoDeclarationHandler` implementations...at least those that do not also declare themselves `first`, and the JetBrains-bundled ones do not.